psi4/driver/procrouting/response/scf_response.py

```diff
@@ -534,54 +559,68 @@ def tdscf_excitations(wfn,
 
     # collect results
     solver_results = []
-    for i, (E_ex_au, irrep_GS, irrep_ES, irrep_trans, edtm_length, f_length, edtm_velocity, f_velocity, mdtm, R_length,
-            R_velocity, spin_mult) in enumerate(_results):
-        sym_descr = f"{irrep_GS}->{irrep_ES} ({1 if spin_mult== 'singlet' else 3} {irrep_trans})"
+    for i, x in enumerate(_results):
+        sym_descr = f"{x.irrep_GS}->{x.irrep_ES} ({1 if x.spin_mult== 'singlet' else 3} {x.irrep_trans})"
 
-        E_ex_ev = constants.conversion_factor('hartree', 'eV') * E_ex_au
+        E_ex_ev = constants.conversion_factor('hartree', 'eV') * x.E_ex_au
 
-        E_tot_au = wfn.energy() + E_ex_au
+        E_tot_au = wfn.energy() + x.E_ex_au
 
         # prepare return dictionary for this root
         solver_results.append({
-            "EXCITATION ENERGY": E_ex_au,
-            "LENGTH-GAUGE ELECTRIC DIPOLE TRANSITION MOMENT": edtm_length,
-            "LENGTH-GAUGE OSCILLATOR STRENGTH": f_length,
-            "VELOCITY-GAUGE ELECTRIC DIPOLE TRANSITION MOMENT": edtm_velocity,
-            "VELOCITY-GAUGE OSCILLATOR STRENGTH": f_velocity,
-            "MAGNETIC DIPOLE TRANSITION MOMENT": mdtm,
-            "LENGTH-GAUGE ROTATORY STRENGTH": R_length,
-            "VELOCITY-GAUGE ROTATORY STRENGTH": R_velocity,
-            "SYMMETRY": irrep_trans,
-            "SPIN": spin_mult,
+            "EXCITATION ENERGY": x.E_ex_au,
+            "LENGTH-GAUGE ELECTRIC DIPOLE TRANSITION MOMENT": x.edtm_length,
+            "LENGTH-GAUGE OSCILLATOR STRENGTH": x.f_length,
+            "VELOCITY-GAUGE ELECTRIC DIPOLE TRANSITION MOMENT": x.edtm_velocity,
+            "VELOCITY-GAUGE OSCILLATOR STRENGTH": x.f_velocity,
+            "MAGNETIC DIPOLE TRANSITION MOMENT": x.mdtm,
+            "LENGTH-GAUGE ROTATORY STRENGTH": x.R_length,
+            "VELOCITY-GAUGE ROTATORY STRENGTH": x.R_velocity,
```

Comment:
I like the dataclass. What do you think for following cc* for the gauge labels?

```
../psi4/src/psi4/cc/ccresponse/optrot.cc:/*- Process::environment.globals["CCSD SPECIFIC ROTATION (VEL) @ xNM"] -*/
../psi4/src/psi4/cc/ccresponse/optrot.cc:/*- Process::environment.globals["CCSD SPECIFIC ROTATION (LEN) @ xNM"] -*/
```

Reply:
Much better, since there's less to type.